lib/deployinator/controller.rb

```diff
@@ -107,10 +107,12 @@ def run(options)
       deploy_instance = deploy_class.new(options)
       deploy_instance.register_plugins(options[:stack])
 
-      locked = deploy_instance.lock_pushes(options[:stack], options[:username], options[:method])
+      if requires_lock
+        locked = deploy_instance.lock_pushes(options[:stack], options[:username], options[:method])
```

Comment:
seems like we should initialized `locked` above this since its now scoped to inside the if, and then referenced below?